provisioner/roles/manage_ec2_instances/defaults/main.yml

```diff
@@ -70,3 +69,19 @@ ec2_info:
     architecture: x86_64
     filter: 'F5 Networks Prelicensed Hourly BIGIP-13.1*Good 25MBPS*'
     username: admin
+  splunk_enterprise:
+    owners: 309956199498
+    size: t2.micro
```

Comment:
Immediately upon login I got a warning about running out of space. We should pick something larger, for example a c4.4xlarge. See the [Splunk AWS tech brief](https://www.splunk.com/pdfs/technical-briefs/deploying-splunk-enterprise-on-amazon-web-services-technical-brief.pdf): that is of course for production usage but recommends even larger instances.

Reply:
@liquidat +1 - I'm like 90% sure I meant to do that and then just didn't